--- scriptMaj	2010-08-30 15:16:48.075000187 +0200
+++ scriptMaj_no_lafiefixer_layman	2010-08-30 15:19:05.677000184 +0200
@@ -60,22 +60,6 @@ color_blue()
 
 color_blue "Update Portage Tree And Layman's Overlays"
 
-layman_sync()
-{
-    if which layman > /dev/null 2>&1 ; then
-        color_green "Update Layman's Overlays"
-        layman -S || color_red "Failed to Update Layman"
-    fi
-}
-
-layman_list()
-{
-    if which layman > /dev/null 2>&1 ; then
-        color_green "List Layman's Overlays"
-        layman -L || color_red "Failed to list Layman"
-    fi
-}
-
 eix_sync()
 {
     if which eix-sync > /dev/null 2>&1 ; then
@@ -108,14 +92,6 @@ emerge_world()
     fi
 }
 
-lafile_fixer()
-{
-    if which lafilefixer > /dev/null 2>&1 ; then
-        color_green "Run lafilefixer"
-        lafilefixer --justfixit || color_red "Failed to run lafilefixer"
-    fi
-}
-
 revdep_rebuild()
 {
     if which revdep-rebuild > /dev/null 2>&1 ; then
@@ -125,7 +101,7 @@ revdep_rebuild()
 }
 
 # Menu for interactive update
-function_update="layman_sync layman_list eix_sync emerge_portage fix_package emerge_world lafile_fixer revdep_rebuild"
+function_update="eix_sync emerge_portage fix_package emerge_world revdep_rebuild"
 
 for i in $function_update ; do
     color_blue "Your are on menu for update your Gentoo/Linux, enter yes or no for update $i"
