diff -urN gmediafinder.orig/setup.py gmediafinder/setup.py
--- gmediafinder.orig/setup.py	2011-06-19 15:37:44.000000000 +0200
+++ gmediafinder/setup.py	2011-06-19 23:42:40.398891290 +0200
@@ -113,17 +113,5 @@
 				break
 			i += 1
 
-	if not prefix:
-		prefix = '/usr'
-	os.system('chmod +x %s' % os.path.join(prefix,'bin/gmediafinder'))
-	gtk_update_icon_cache = '''gtk-update-icon-cache -f -t \
-%s/share/icons/hicolor''' % prefix
-	root_specified = [s for s in sys.argv if s.startswith('--root')]
-	if not root_specified or root_specified[0] == '--root=/':
-		print 'Updating Gtk icon cache.'
-		os.system(gtk_update_icon_cache)
-	else:
-		print '''*** Icon cache not updated. After install, run this:
-***     %s''' % gtk_update_icon_cache
         os.system('xdg-desktop-menu install --novendor gmediafinder.desktop')
 
