--- a/misc/CMakeLists.txt	2017-07-26 11:23:12.576297648 +0200
+++ b/misc/CMakeLists.txt	2017-07-26 11:26:23.311188725 +0200
@@ -138,7 +138,7 @@
   else (MINGW)
     target_link_libraries (openvas_misc_shared ${GNUTLS_LDFLAGS} openvas_base_shared ${UUID_LDFLAGS}
                            ${PCAP_LDFLAGS} ${LIBSSH_LDFLAGS} ${LDAP_LDFLAGS}
-                           ${REDIS_LDFLAGS} ${GCRYPT_LDFLAGS} ${RADIUS_LDFLAGS})
+                           ${GLIB_LDFLAGS} ${REDIS_LDFLAGS} ${GCRYPT_LDFLAGS} ${RADIUS_LDFLAGS})
   endif (MINGW)
 endif (BUILD_SHARED)
 
