--- a/base/CMakeLists.txt	2017-07-26 11:22:43.152314452 +0200
+++ b/base/CMakeLists.txt	2017-07-26 11:24:46.271244142 +0200
@@ -89,7 +89,7 @@
 # TODO: This is currently only built for BUILD_SHARED
 if (BUILD_SHARED)
   add_executable (test-hosts test-hosts.c)
-  target_link_libraries (test-hosts openvas_base_shared ${GNUTLS_LDFLAGS} openvas_misc_shared ${GLIB_LDFLAGS})
+  target_link_libraries (test-hosts openvas_base_shared ${GNUTLS_LDFLAGS} openvas_misc_shared openvas_omp_shared ${GLIB_LDFLAGS})
 endif (BUILD_SHARED)
 
 ## Install
