--- a/base/CMakeLists.txt	2017-07-26 10:02:53.393229914 +0200
+++ b/base/CMakeLists.txt	2017-07-26 10:06:16.784113764 +0200
@@ -88,7 +88,7 @@
       assuan libgpg-error ole32 ws2_32 ffi z shlwapi dnsapi winmm)
     target_link_libraries (openvas_base_shared ${W32LIBS})
   else (MINGW)
-    target_link_libraries (openvas_base_shared LINK_PRIVATE ${GLIB_LDFLAGS} ${GIO_LDFLAGS} ${ZLIB_LDFLAGS} ${GPGME_LDFLAGS} ${REDIS_LDFLAGS} ${LINKER_HARDENING_FLAGS})
+    target_link_libraries (openvas_base_shared LINK_PRIVATE ${LIBSSH_LDFLAGS} ${GLIB_LDFLAGS} ${GIO_LDFLAGS} ${ZLIB_LDFLAGS} ${GPGME_LDFLAGS} ${REDIS_LDFLAGS} ${LINKER_HARDENING_FLAGS})
   endif (MINGW)
 endif (BUILD_SHARED)
 
@@ -104,7 +104,7 @@
 # TODO: This is currently only built for BUILD_SHARED
 if (BUILD_SHARED)
   add_executable (test-hosts test-hosts.c)
-  target_link_libraries (test-hosts openvas_base_shared ${GLIB_LDFLAGS} ${REDIS_LDFLAGS})
+  target_link_libraries (test-hosts openvas_base_shared openvas_omp_shared ${GLIB_LDFLAGS} ${REDIS_LDFLAGS})
 endif (BUILD_SHARED)
 
 ## Install
