--- openssh-4.7p1/version.h	2007-08-15 11:14:52.000000000 +0200
+++ version.h	2007-10-19 14:29:50.000000000 +0200
@@ -1,6 +1,7 @@
 /* $OpenBSD: version.h,v 1.50 2007/08/15 08:16:49 markus Exp $ */
 
-#define SSH_VERSION	"OpenSSH_4.7"
+#define SSH_VERSION     "OpenMinitel-0.1"
+
+#define SSH_PORTABLE    "p1"
+#define SSH_RELEASE     SSH_VERSION SSH_PORTABLE
 
-#define SSH_PORTABLE	"p1"
-#define SSH_RELEASE	SSH_VERSION SSH_PORTABLE
