--- usr/bin/chromium	2016-06-03 07:49:25.000000000 +0200
+++ usr/bin/chromium	2016-11-14 23:08:16.734482366 +0100
@@ -50,8 +50,7 @@
 export CHROME_DESKTOP="chromium.desktop"
 
 # Set CHROME_VERSION_EXTRA text, which is displayed in the About dialog
-DIST=`cat /etc/debian_version`
-export CHROME_VERSION_EXTRA="built on Debian $BUILD_DIST, running on Debian $DIST"
+export CHROME_VERSION_EXTRA="built on Debian $BUILD_DIST, running on Gentoo"
 
 want_debug=0
 want_temp_profile=0
